db/schema.rb

```diff
@@ -0,0 +1,8 @@
+ActiveRecord::Schema.define(version: 20240422130000) do
+
+  create_table 'users', id: { type: :integer, comment: 'ユーザーID', unsigned: true, default: nil }, charset: 'utf8mb4', collation: 'utf8mb4_bin', comment: 'ユーザー', force: :cascade do |t|
```

Comment:
PKとなるIDは、自動で付与されると思うので、以下の記述でいいと思う！


```suggestion
  create_table 'users', charset: 'utf8mb4', collation: 'utf8mb4_bin', comment: 'ユーザー', force: :cascade do |t|
```

Reply:
[f400c29](https://github.com/ok-os-job-change-team/tetsuya-twitter-clone-bootcamp/pull/21/commits/f400c293e24e606c4dda3ca455b41dc64c1ad4a1)
で対応